gateway/enforcer/org.wso2.choreo.connect.enforcer.commons/src/main/java/org/wso2/choreo/connect/enforcer/commons/model/EndpointSecurity.java

```diff
@@ -16,42 +16,87 @@
  * under the License.
  */
 package org.wso2.choreo.connect.enforcer.commons.model;
+import java.util.Map;
 
 /**
  * EndpointSecurity holds the EndpointSecurity configuration for an API.
  */
 public class EndpointSecurity {
```

Comment:
There's SecurityInfo model duplicating this. Use it after refactoring

Reply:
removed SecurityInfo model and kept this, if not we need to remove EndpointSecurity model since now both do same thing